extra_data/tests/test_file_access.py

```diff
@@ -87,24 +87,31 @@ def test_euxfel_path_infos(mock_sa3_control_data, monkeypatch):
     for filename in [
         '/gpfs/exfel/exp/SA3/202301/p001234/raw/r0100/foo.h5',
         '/gpfs/exfel/d/raw/SA3/202301/p001234/r0100/foo.h5',
-        '/pnfs/xfel.eu/exfel/archive/XFEL/raw/SA3/202301/p001234/r0100/foo.h5'
+        '/pnfs/xfel.eu/exfel/archive/XFEL/raw/SA3/202301/p001234/r0100/foo.h5',
+        '/gpfs/exfel/exp/SA3/202301/p001234/usr/.extra_data/RAW-R0100-OVERVIEW.h5'
     ]:
         fa = FileAccess(filename, _cache_info=_empty_cache_info)
 
-        assert fa.storage_class == 'raw'
+        assert fa.storage_class in {'raw', 'usr'}
         assert fa.instrument == 'SA3'
         assert fa.cycle == '202301'
+        assert fa.proposal == 1234
```

Comment:
```suggestion
        assert fa.proposal == 1234
        assert fa.run == 5100
```

Reply:
That actually doesn't work, as `run` was only inferred from filename, not path. But we can actually, so I implemented this test by having run be a two-step check.